### PR TITLE
Fix 1.5 telemetry v2 config

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

File: istio-control/istio-discovery/templates/telemetryv2_1.5.yaml [hunk 1]
```diff
@@ -124,13 +121,13 @@ spec:
                 {
                   "debug": "false",
                   "stat_prefix": "istio",
+                  "disable_host_header_fallback": true
```
Comment (kyessenov) [inline istio-control/istio-discovery/templates/telemetryv2_1.5.yaml]: I think we may want this.

Comment (howardjohn) [reply istio-control/istio-discovery/templates/telemetryv2_1.5.yaml]: Does want this mean `true` or `false`?

**Comment by richardwxn** [comment]
Are you using the templates from istio/proxy? But the one from istio tests should be the source of truth? https://github.com/istio/istio/blob/master/tests/integration/telemetry/stats/prometheus/testdata/stats_filter.yaml

Need confirmation from @bianpengyuan @douglas-reid @mandarjog

**Comment by howardjohn** [comment]
I copied from istio/proxy. it would be nice to not have this in 4 different places

the main change here is to the metadata exchange the other part is just formatting to match the other examples

**Pull request closed without merging**